tests/manifests/migrator.yaml

```diff
@@ -3,24 +3,24 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: rook-ceph-tools
+  name: rook-ceph-migrator
   namespace: rook-ceph # namespace:cluster
   labels:
-    app: rook-ceph-tools
+    app: rook-ceph-migrator
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: rook-ceph-tools
+      app: rook-ceph-migrator
   template:
     metadata:
       labels:
-        app: rook-ceph-tools
+        app: rook-ceph-migrator
     spec:
-      serviceAccountName: rook-ceph-tools # only this field was newly added
+      serviceAccountName: rook-ceph-migrator # only this field was newly added
       dnsPolicy: ClusterFirstWithHostNet
       containers:
-        - name: rook-ceph-tools
+        - name: rook-ceph-migrator
           image: rook/ceph:v1.7.5
           command: ["/tini"]
```

Comment:
There is no more `tini` in the rook image, see https://github.com/rook/rook/pull/8805 for similar changes we need here as well. Could be a separate PR though...

Reply:
Since,`tini` remove PR merged in master and flex is removed from the master. So, I was using all the required files from rook v1.7.x only. But, I can update in follow up PR soon.

Reply:
#26

Reply:
Yeah I realized on #26 that we don't need that change